### ISSUE TITLE
Purpose/why for preemptible_and_regular_instance_templates module

### ISSUE THREAD

**Issue by mafrosis** [terraform-google-modules/terraform-google-vm#101]
A five minute click around and read of the docs suggests that [`preemptible_and_regular_instance_templates`](https://github.com/terraform-google-modules/terraform-google-vm/tree/master/modules/preemptible_and_regular_instance_templates) module exists because the normal `instance_template` module can be only preemptible _or_ regular, but not both.

Is this correct? I would suggest it's worth adding "why would I use this" to all the modules README very close to the top 😄

**Comment by morgante** [comment]
Yes, the idea is to be able to have a workload defined once but deployed using a mix of preemptible and regular VMs.

Agreed that we can add more value/usage info to the README. Happy to accept a PR.
